generators/chipyard/src/main/scala/config/SpikeConfigs.scala

```diff
@@ -0,0 +1,19 @@
+package chipyard
+
+import freechips.rocketchip.config.{Config}
+
+// Configs which instantiate a Spike-simulated
+// tile that interacts with the Chipyard SoC
+// as a hardware core would
+
+class SpikeConfig extends Config(
+  new chipyard.WithNSpikeCores(1) ++
+  new chipyard.config.AbstractConfig)
+
+// Avoids polling on the UART registers
+class SpikeFastUARTConfig extends Config(
+  new chipyard.WithNSpikeCores(1) ++
+  new chipyard.config.WithUARTFIFOEntries(128, 128) ++
+  new chipyard.config.WithMemoryBusFrequency(1) ++
+  new chipyard.config.WithPeripheryBusFrequency(1) ++
```

Comment:
Why is this config nice/needed? For Linux how much speedup is obtained?

Reply:
Nit: Can you order the fragments with the number of cores at the bottom (similar to how the Boom/Rocket configs are setup)

Reply:
This makes the UART run at a similar frequency to the uncore, otherwise the UART runs super slowly compare to the core, and the core gets stuck behind polling UART.